server/ntb/io/feed_parsers/dpa_newsml.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2018 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.io.feed_parsers.newsml_2_0 import NewsMLTwoFeedParser
+from superdesk.io.registry import register_feed_parser
+from superdesk.errors import ParserError
+from superdesk.metadata.item import CONTENT_TYPE
+from superdesk import etree as sd_etree
+from lxml import etree, html
+from lxml.html import clean
+from . import utils
+import dateutil.parser
+import logging
+
+logger = logging.getLogger(__name__)
+NS = {'xhtml': 'http://www.w3.org/1999/xhtml',
+      'iptc': 'http://iptc.org/std/nar/2006-10-01/'}
+
+
+class DPANewsMLFeedParser(NewsMLTwoFeedParser):
+    """
+    Feed Parser which can parse DPA variant of NewsML
+    """
+
+    NAME = 'dpanewsml'
+    label = "DPA NewsML"
+
+    def can_parse(self, xml):
+        return xml.tag.endswith('newsMessage')
+
+    def parse(self, xml, provider=None):
+        self.root = xml
+        items = []
+        try:
+            header = self.parse_header(xml)
+            for item_set in xml.findall(self.qname('itemSet')):
+                for item_tree in item_set:
+                    item = self.parse_item(item_tree)
+                    item['versioncreated'] = header['sent']
```

Comment:
why using sent here? that's not the same thing as versioncreated

Reply:
I though it was. Should I just use time of parsing for versioncreated?

Reply:
it's in [itemMeta](https://github.com/superdesk/superdesk-ntb/pull/210/files#diff-2bab21f8d232eb33a0c1c586c64cdb13R30)